drools-wb-screens/drools-wb-globals-editor/drools-wb-globals-editor-backend/src/test/java/org/drools/workbench/screens/globals/backend/server/GlobalsEditorServiceTest.java

```diff
@@ -93,7 +92,7 @@ public void save() {
         verify(ioService,
                times(1)).write(any(org.uberfire.java.nio.file.Path.class),
                                anyString(),
-                               any(CommentedOption.class));
+                               any());
```

Comment:
@caponetto Ok, I think I can stop to use CTRL+C and simply say: Please check all usage of `any()` you added and evaluate if substitute it with `isA(Clazz.class)`. :)